util/clock/timesource.go

```diff
@@ -106,18 +110,19 @@ func (f *FakeTimeSource) Set(t time.Time) {
 	}
 }
 
-// IncrementingFakeTimeSource takes a base time and several increments, which will be applied to
-// the base time each time Now() is called. The first call will return the base time + zeroth
-// increment. If called more times than provided for then it will panic. Does not require that
-// increments increase monotonically.
+// IncrementingFakeTimeSource takes a base time and several increments, which
+// will be applied to the base time each time Now() is called. The first call
+// will return the base time + zeroth increment. If called more times than
+// provided for then it will panic. Does not require that increments increase
+// monotonically.
 type IncrementingFakeTimeSource struct {
```

Comment:
Can probably shorten this name as well now. Can be done later.

Reply:
Done. Please review the last commit.